src/version.rc

```diff
@@ -0,0 +1,39 @@
+#include <verrsrc.h>
+
+// A PowerShell script replaces the tokens with actual version information
+
+VS_VERSION_INFO VERSIONINFO
+ FILEVERSION V8JSIVER_MAJOR,V8JSIVER_MINOR,V8JSIVER_BUILD,0
+ PRODUCTVERSION V8JSIVER_MAJOR,V8JSIVER_MINOR,V8JSIVER_BUILD,0
+ FILEFLAGSMASK 0x17L
+#ifdef _DEBUG
+ FILEFLAGS 0x1L
+#else
+ FILEFLAGS 0x0L
+#endif
+ FILEOS 0x4L
+ FILETYPE 0x1L
+ FILESUBTYPE 0x0L
+BEGIN
+    BLOCK "StringFileInfo"
+    BEGIN
+        BLOCK "040904b0"
+        BEGIN
+            VALUE "CompanyName", "Microsoft"
+            VALUE "FileDescription", "React Native V8 JSI Adapter"
+            VALUE "FileVersion", "V8JSIVER_MAJOR.V8JSIVER_MINOR.V8JSIVER_BUILD.V8JSIVER_V8REF"
+            VALUE "InternalName", "v8jsi.dll"
+            VALUE "LegalCopyright", "(c) Microsoft Corporation. All rights reserved."

```

Comment:
>All rights reserved [](start = 64, length = 19)

Can you double check license?

For MIT license, this is not recommended

[https://docs.opensource.microsoft.com/changelog.html?q=all%20rights%20reserved](https://docs.opensource.microsoft.com/changelog.html?q=all%20rights%20reserved)
July 2019
CELA open source policy
The term All Rights Reserved is no longer recommended to be in MIT license copyright headers.

Reply:
Great catch!

---
In reply to: [432033949](https://github.com/microsoft/v8-jsi/pull/14#discussion_r432033949) [](ancestors = 432033949)